DQM/L1TMonitor/src/L1TdeRCT.cc

```diff
@@ -122,20 +123,6 @@ L1TdeRCT::L1TdeRCT(const ParameterSet & ps) :
   if (verbose_)
     std::cout << "L1TdeRCT: constructor...." << std::endl;
 
-  outputFile_ =
-      ps.getUntrackedParameter < std::string > ("outputFile", "");
```

Comment:
If this parameter is not used anymore it should also be removed from the config files.